src/pages/chainlink-nodes/v1/fulfilling-requests.md

```diff
@@ -33,15 +33,15 @@ Before you begin this guide, complete the following tasks to make sure you have
 
 - [Set up MetaMask](/getting-started/deploy-your-first-contract/#install-and-fund-your-metamask-wallet) and [obtain testnet LINK](/resources/acquire-link/).
 - [Run a Chainlink Node](/chainlink-nodes/v1/running-a-chainlink-node).
-- Fund the Ethereum address that your Chainlink node uses. You can find the address in the node Operator GUI under the **Key Management** configuration. The address of the node is the `Regular` type. You can obtain test ETH from several [faucets](/resources/link-token-contracts/). For this tutorial to work, you will have to fund the node's Ethereum address with Goerli ETH. Here is an example:
+- Fund the Ethereum address that your Chainlink node uses. You can find the address in the node Operator GUI under the **Key Management** configuration. The address of the node is the `Regular` type. You can obtain test ETH from several [faucets](/resources/link-token-contracts/). For this tutorial to work, you will have to fund the node's Ethereum address with Sepolia ETH. Here is an example:
 
-  ![chainlink node goerli fund address](/images/chainlink-nodes/node-operators/key-management.jpg)
+  ![chainlink node Goerli fund address](/images/chainlink-nodes/node-operators/key-management.jpg)
```

Comment:
I'm assuming here it looks the same whether it's Sepolia or Goerli, so the image doesn't need an update.

Reply:
I'm grabbing an updated screenshot. I need to do some testing on the latest node versions anyway. I just left this for last or it can be done afterwards.